tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -240,7 +240,7 @@
             </class>
             <class name="org.ballerinalang.test.bala.record.OpenRecordTypeReferenceTest" />
             <class name="org.ballerinalang.test.bala.record.ClosedRecordTypeReferenceTest" />
-            <class name="org.ballerinalang.test.ParserTestRunner"/>
+<!--            <class name="org.ballerinalang.test.ParserTestRunner"/>-->
```

Comment:
Shall we create an issue with Type/TestFailture to track these disabled tests?

Reply:
created #29374